### ISSUE TITLE
Comment on "essential"

### ISSUE THREAD

**Issue by kerstinp** [w3c/wcag21#408]
I'm a bit worried because of "essential". The number of SC which contains "essential" increased from 5 SC in WCAG 2.0 to – if counted correctly – 11 SC in WCAG 2.1 which could cause (more) time-consuming discussions for developers and evaluators as well. 

I want to give just one example where "essential" could be critical: 

SC 1.4.11 says "except a partical presentation of the graphical is essential to the information conveyed". 

One could argue that a presentation of data as a diagramm is always essential because visualizations are better for the understanding of the content for many people or one could say that diagramms for the presentation of data instead of using a data table is never essential. 

In case we have two presentations of the same data: a diagramm and a data table I would say that if the data table pass also SC 1.4.11 pass. But for some people the diagramm might still be "essential" for a better understanding. 

I think it would be better having something like "images which are informative". This would adress the specific character of a graphical content and not the (visual) presentation. And I think it would be much easier for harmonization with Sufficient Techniques and Failures.

**Comment by awkawk** [comment]
Thank you for the comment. The use of "essential" in WCAG 2.1 is needed in some of the new SC as a way to convey an exception for content that must be presented in a specific manner and it is useful to make consistent use of the defined term for this purpose. In the SC identified the term essential refers to the specific way a graphic is presented, rather than the graphic itself being essential. We will clarify this in the understanding document.

"Essential" is a defined term in WCAG, meaning "if removed, would fundamentally change the information or functionality of the content, and information and functionality cannot be achieved in another way that would conform".
